storage/persistent_storage/persistent-storage-ocs.adoc

```diff
@@ -8,6 +8,8 @@ Red Hat OpenShift Container Storage is a provider of agnostic persistent storage
 
 Red Hat OpenShift Container Storage provides its own documentation library. The complete set of Red Hat OpenShift Container Storage documentation identified below is available at https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.6/
 
+OpenShift Container Storage on top of Red Hat Hyperconverged Infrastructure (RHHI) for Virtualization, which uses hyperconverged nodes that host virtual machines installed with {product-title}, is not a supported configuration.
```

Comment:
@Erin-Donnelly - Per comment from @bkunal, you can add after this sentence:
For more information about supported platforms, see the [Red Hat OpenShift Container Storage Supportability and Interoperability Guide](https://access.redhat.com/articles/4731161).